src/AddTodoForm.module.css

```diff
@@ -0,0 +1,40 @@
+.form {
+  display: flex;
+  width: 100%;
+  max-width: 550px;
+  justify-content: space-between;
+  min-height: 1rem;
+  align-items: center;
+  gap: 10px;
+}
+
+.addButton {
+  border-radius: 8px;
+  border-color: transparent;
+  border: 0;
+  width: auto;
+  max-width: 90px;
+  background-color: #7984e6;
+  cursor: pointer;
+  padding: 12px;
+  margin-right: 3px;
+  margin-left: 5px;
+  width: 25%;
+  font-weight: bold;
+}
+
+  @media (max-width: 480px) {
+   .form {
+    flex-direction: column;
+    align-items: center;
+    gap: 1px;
+  }
+}
+
+@media (min-width: 768px) {
+  .form {
+    flex-direction: row;
+    width: 100%;
```

Comment:
you already has width for .form: 100% on the line 3. Do you need the same code here?

Reply:
Removed!👍🏻